### PR TITLE
Limit carrier image size on checkout page

### LINKED ISSUE #9691: [BOOM-5190] Logo of a carrier in checkout page not limited
> This issue has been migrated from the Forge. Read the [original ticket here](http://forge.prestashop.com/browse/BOOM-5190).

- _**Reporter:**_ khouloud.belguith
- _**Created at:**_ Mon, 26 Mar 2018 19:14:12 +0200

<p>Hi,</p>

<p>In the checkout page, the logo of a carrier is not limited<br/>
I added a logo which has a size too big.<br/>
It's displayed wrongly on the checkout page, see the image.<br/>
I think it should be limited.</p>

<p>Best regards, Khouloud</p>
        
* How to reproduce the issue ?

N/A

### PR DESCRIPTION
**Pull request by YeLnatSs**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | limit size for carrier image on checkout page.
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | Does it break backward compatibility? no
| Deprecations? | Does it deprecate an existing feature? no
| Fixed ticket? | Fixes #9691 
| How to test?  | Buy a product and go to checkout page. On shipping method, you'll be able to see carrier image has a limit size.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/12892)

### CODE REVIEW

File: themes/classic/_dev/css/components/checkout.scss [hunk 1]
```diff
@@ -222,6 +222,9 @@ body#checkout {
       label {
         text-align: inherit;
       }
+      img{
```
Comment (PierreRambaud) [inline themes/classic/_dev/css/components/checkout.scss]: Can you add a space before `{` please ? :smile:

Comment (YeLnatSs) [reply themes/classic/_dev/css/components/checkout.scss]: It's done

**Review by eternoendless — DISMISSED** [review]

**Comment by marionf** [comment]
Removing QA approved, because the issue isn't completely fixed

@YeLnatSs The issue is also [on the payment step](https://github.com/PrestaShop/PrestaShop/issues/9691#issuecomment-415075911) when the final summary in Shop parameters > order settings is enabled

capture d'écran_1154

**Comment by YeLnatSs** [comment]
@marionf I fixed the problem and add a new logo for carrier and a new size according to @TristanLDD

**Comment by marionf** [comment]
Thank you @YeLnatSs 
Could you please resolve conflicts ?
I will retest it when code review will be ok

**Comment by YeLnatSs** [comment]
Hello @marionf can you check it again?

**Comment by marionf** [comment]
It's all good @YeLnatSs

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @YeLnatSs

**Comment by ProBackup-nl** [comment]
Thanks for the patch.
In 1.7.5.2 there is no `checkout.scss`, not even a `checkout.css`.
In the mean time, where to manually patch this issue carrier image size issue in **1.7.5.2**?

My best guess is to edit `./themes/classic/assets/css/theme.css`
Search for `body#checkout section.checkout-step .delivery-option label{text-align:inherit}` and insert after that:
```
body#checkout section.checkout-step .delivery-option img{width:3.125rem}
```

Secondly search for `body#checkout #order-summary-content .summary-selected-carrier{`
and after that statement insert:
```
body#checkout #order-summary-content .summary-selected-carrier .logo-container{text-align:center}
body#checkout #order-summary-content .summary-selected-carrier .logo-container img{width:3.125rem}
```

**Pull request merged**